pandas/io/parsers.py

```diff
@@ -1582,6 +1581,7 @@ def __init__(self, f, **kwds):
         self.converters = kwds['converters']
 
         self.thousands = kwds['thousands']
+        self.decimal = kwds['decimal']
```

Comment:
pls check / update ``io.rst`` and the doc-string. IIRC we list in the option if it doesn't support a particular engine (so that can now be removed).